Missingreferences/Editor/MissingSceneReferences.cs

```diff
@@ -7,6 +8,15 @@ namespace Unity.Labs.SuperScience
 {
     sealed class MissingSceneReferences : MissingReferencesWindow
     {
+        const string k_Instructions = "Click the Refresh button to scan the active scene for missing references. WARNING: " +
+            "For large scenes, this may take a long time and/or crash the Editor.";
+
+        const string k_NoMissingReferences = "No missing references in active scene";
+
+        // Bool fields will be serialized to maintain state between domain reloads, but our list of GameObjects will not
+        [NonSerialized]
```

Comment:
Should this be SerializeField?

Reply:
No. I guess I should make the comment more clear? We _don't_ want this serialized between domain reloads.